practice/flutter/flutter/views.py

```diff
@@ -0,0 +1,60 @@
+"""flutter Views."""
+
+from . import logic
+from . import models
+from django.shortcuts import render
+
+
+def flutts_by_time(flutts):
+    """Takes in a series of flutt objects, and orders them by time with the most recent first."""
+    return flutts.order_by('timestamp').reverse()
```

Comment:
This function feels like a logic function. It doesn't have anything directly to do with the HTTP interface of this app. It also can be doctested.